custom-css-analysis-and-migration/create-migrations.ts

```diff
@@ -36,21 +36,47 @@ for (const record of records) {
   if (record.css === undefined) continue;
 
   ++count;
-  if (Bun.argv.length > 2 && count >= Number.parseInt(Bun.argv[2])) break;
+  if (Bun.argv.length > 2 && count > Number.parseInt(Bun.argv[2])) break;
   console.log(`css = ${record.css}`);
   const checksum = crypto.createHash("md5").update(record.css).digest("hex");
   const folder = `./output/migration/${count} ${checksum}`;
+  console.log(`output folder=${folder}`);
   fs.mkdirSync(folder);
 
+  let cssPath = `${folder}/customBookStyles.css`;
+  if (fs.existsSync(cssPath)) {
+    cssPath = `${folder}/customBookStyles2.css`;
+    console.log(`WARNING: multiple migrations in ${folder}`);
```

Comment:
This handles the case where both customCollectionStyles.css and customBookStyles.css have css that need to be migrated.  By the time this code runs, the original filename is lost, so I compromised by providing two different output files.  The migration code erases any earlier output when it starts, so a duplicate has to happen in this run.
It's quite possible I've been overly paranoid in this instance, and the problem never arises in practice.

Reply:
Now that I think about it, I think this situation never arises in the migration output folder because the migration is specific to a single file matching a specific checksum.  So this code can be simplified here.